renovate.json

```diff
@@ -1,15 +1,50 @@
 {
-  "extends": [
-    "config:base"
-  ],
+  "extends": ["config:base"],
   "dependencyDashboardApproval": true,
-  "labels": ["PR: Dependency ⬆️", "repo automation 🤖" ],
+  "labels": ["PR: Dependency ⬆️", "repo automation 🤖"],
   "postUpdateOptions": ["yarnDedupeHighest"],
+  "regexManagers": [
+    {
+      "fileMatch": ["^Makefile$"],
+      "matchStrings": ["TEST262_COMMIT = (?<currentDigest>.*)\\n"],
+      "datasourceTemplate": "git-refs",
+      "depNameTemplate": "tests-test262",
+      "lookupNameTemplate": "https://github.com/tc39/test262.git",
+      "currentValueTemplate": "main"
+    },
+    {
+      "fileMatch": ["^Makefile$"],
+      "matchStrings": ["FLOW_COMMIT = (?<currentDigest>.*)\\n"],
+      "datasourceTemplate": "git-refs",
+      "depNameTemplate": "tests-flow",
+      "lookupNameTemplate": "https://github.com/facebook/flow.git",
+      "currentValueTemplate": "master"
+    },
+    {
+      "fileMatch": ["^Makefile$"],
+      "matchStrings": ["TYPESCRIPT_COMMIT = (?<currentDigest>.*)\\n"],
+      "datasourceTemplate": "git-refs",
+      "depNameTemplate": "tests-typescript",
+      "lookupNameTemplate": "https://github.com/Microsoft/TypeScript.git",
+      "currentValueTemplate": "main"
+    },
+    {
+      "fileMatch": ["^Makefile$"],
+      "matchStrings": ["COMPAT_TABLE_COMMIT=(?<currentDigest>.*)\\n"],
```

Comment:
This isn't automatically updated by this PR, but it should appear in the dependencies dashboard.